boards/nxp/frdm_ke17z/doc/index.rst

```diff
@@ -61,6 +61,8 @@ features:
 +-----------+------------+-------------------------------------+
 | I2C       | on-chip    | i2c                                 |
 +-----------+------------+-------------------------------------+
+| ACMP      | on-chip    | analog comparator                   |
```

Comment:
the driver/component column should be "sensor"

Reply:
Updated.